extensions/wikia/Forum/ForumDisabled.setup.php

```diff
@@ -8,7 +8,9 @@
 
 $wgExtensionCredits['specialpage'][] = array( 
 	'name' => 'Forum',
-	'author' => array( 'Kyle Florence', 'Saipetch Kongkatong', 'Tomasz Odrobny' )
+	'author' => array( 'Kyle Florence', 'Saipetch Kongkatong', 'Tomasz Odrobny' ),
+	'url' => 'https://github.com/Wikia/app/tree/dev/extensions/wikia',
```

Comment:
I imagine this should also be `https://github.com/Wikia/app/tree/dev/extensions/wikia/Forum`.